tests/backup/backup_helper.go

```diff
@@ -13,22 +13,25 @@ import (
 	"sync"
 	"time"
 
-	"github.com/portworx/sched-ops/k8s/apps"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Why are we moving it down what's the advantage of doing this?

Reply:
This lines are moved as part of go auto-formatting.